server/tests/api/videos/video-transcoder.ts

```diff
@@ -281,7 +282,67 @@ describe('Test video transcoding', function () {
     }
   })
 
+  it('Should respect maximum bitrate values', async function () {
+    this.timeout(80000)
+
+    {
+      // Generate a random, high bitrate video on the fly, so we don't have to include
+      // a large file in the repo.
+      // https://stackoverflow.com/a/15795112
+      await new Promise<string>((res, rej) => {
+        ffmpeg()
+          .outputOptions(['-f rawvideo', '-video_size 1920x1080', '-i /dev/urandom'])
+          .outputOptions(['-ac 2', '-f s16le', '-i /dev/urandom', '-vframes 3'])
+          .output(buildAbsoluteFixturePath('video_high_bitrate_1080p.mp4'))
+          .on('error', rej)
+          .on('end', () => res)
+          .run()
+      })
+
+      const videoAttributes = {
+        name: 'waiting video',
+        fixture: 'video_high_bitrate_1080p.mp4',
+        waitTranscoding: false
+      }
+
```

Comment:
I suggest to ensure your generated file has a higher bitrate than `getMaxBitrate`